package.json

```diff
@@ -128,7 +128,7 @@
   },
   "scripts": {
     "postinstall": "npm rebuild node-sass",
-    "prepublishOnly": "yarn run build && yarn run test:noPrompt",
+    "prepublishOnly": "yarn install && run build && yarn run test:noPrompt",
```

Comment:
Is this command the same as `yarn install && yarn publish` you mentioned to me during our meeting?

Reply:
We sorted it out. Thanks Michael!